build/test.mk

```diff
@@ -58,7 +58,7 @@ test-get-envtools:
 test-validate-cli: ## Run cli integration tests
 	CGO_ENABLED=1 $(GOTEST_TOOL) -coverpkg= ./pkg/cli/cmd/... ./cmd/rad/... -timeout ${TEST_TIMEOUT} -v -parallel 5 $(GOTEST_OPTS)
 
-test-functional-all: test-functional-ucp test-functional-kubernetes test-functional-shared test-functional-msgrp test-functional-daprrp ## Runs all functional tests
```

Comment:
Why are you taking this out? `test-functional-kubernetes` has one test 👀

Reply:
If tests are being purposely disabled, create an issue and leave a reference to it here in a comment.

Reply:
adding it back in since it was only disabled for the long running test